From a6747362693a91f8cfa3bd5e619cf91e2e03941b Mon Sep 17 00:00:00 2001
From: David Zeuthen <zeuthen@gmail.com>
Date: Thu, 15 Nov 2012 10:09:06 -0500
Subject: [PATCH] Don't require libmozjs185 devel packages for polkit rules to
 work

The SO-name rules for SpiderMonkey are a little bit weird and we were
opening wrong file. This problem was reported in

 https://bugs.freedesktop.org/show_bug.cgi?id=57146

Signed-off-by: David Zeuthen <zeuthen@gmail.com>
---
 src/polkitbackend/polkitbackendjsauthority.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/polkitbackend/polkitbackendjsauthority.c b/src/polkitbackend/polkitbackendjsauthority.c
index 034edc4..e4aa46b 100644
--- a/src/polkitbackend/polkitbackendjsauthority.c
+++ b/src/polkitbackend/polkitbackendjsauthority.c
@@ -238,7 +238,7 @@ djs_init (PolkitBackendJsAuthority *authority)
   guint n;
   const gchar *library_name;
 
-  library_name = "libmozjs185.so";
+  library_name = "libmozjs185.so._MOZJSSOVER_";
   module = g_module_open (library_name, 0);
   if (module == NULL)
     goto out;
-- 
1.8.0

